File: tasks/2024/round3_pp/csoki2.cpp
// @check-accepted: examples N=0 no-limits

#include <fstream>
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int M, N, K;

    cin >> M >> N >> K;

    if ( (N+M)%K == 0 ) {
        cout << "IGEN" << endl;
    } else {
        cout << "NEM" << endl;
      }

    return 0;
}

File: tasks/2024/round3_pp/csoki2.py
#!/usr/bin/env python3
# @check-accepted: examples N=0 no-limits



# input data
M,N,K = list(map(int, input().strip().split()))


# insert your code here

if (N+M)%K == 0:
    print('IGEN')  # print the result
else:
    print('NEM')  # print the result
